pitch.md

```diff
@@ -1,44 +1,80 @@
-# Project Title (replace me)
+# Shared Spaces
 
 ## Overview
 _What's this project about, briefly?_
 
+This project serves a group of people who share a set of spaces and need to use the spaces at various times. Examples include kitchens, lab spaces, and study room. This language would ideally be good for any user to reserve a space or designate the space as 'occupied'.
```

Comment:
This strikes me as both a language to be designed and a service to be provided. Will you be developing both in your project?